test/Suites.scala

```diff
@@ -0,0 +1,7 @@
+import integration.IntegrationTests
+import org.scalatest.Suites
+import test.ThriftUtilSpec
+
+class UnitTestsSuite extends Suites(new ThriftUtilSpec)
```

Comment:
Solid idea, will see if I can get regular 'sbt test' to ignore the integration tests by default to make local testing easier too